src/lib/openapi/spec/client-features-query-schema.ts

```diff
@@ -4,30 +4,46 @@ export const clientFeaturesQuerySchema = {
     $id: '#/components/schemas/clientFeaturesQuerySchema',
     type: 'object',
     additionalProperties: false,
+    description: 'Query parameters active for a client features request',
     properties: {
         tag: {
             type: 'array',
+            description: 'Features tagged with one of these tags are included',
```

Comment:
Hmm maybe something like this to make it clearer that it excludes features without the tags?

```suggestion
            description: 'Only return features tagged with at least one of the provided tags',
```